JobConfig/beam/oot.fcl

```diff
@@ -18,7 +18,9 @@ physics.producers.generate: {
     module_type: StoppedParticleG4Gun
     verbosityLevel : 1
     pdgId: 13 // mu-
-    muonStops : @local::mu2e.ootMuonStops
+    muonStops : { @table::mu2e.ootMuonStops
+   	       	  inputFiles : @local::mergedMuon_ootStops_mdc2018
```

Comment:
Same for OOT: this replaces @nil with a hardcoded set of input files.   I think this change is outside the scope of this PR, please take it out.

Reply:
Done (commit 3e76dc9e4)